charts/tfy-jspolicy-configs/templates/pod-volume-mounts.yaml

```diff
@@ -38,13 +38,13 @@ spec:
     // add the volumes to the pod
     const podVolumes = request.object.spec.volumes || [];
     mountDetails.forEach(mountDetails => {
```

Comment:
We should rename the each item - both the list and item are being called `mountDetails`

This can cause confusion